tests/validation/tests/v3_api/common.py

```diff
@@ -162,6 +162,10 @@
 # the shared password for all auth users
 AUTH_USER_PASSWORD = os.environ.get('RANCHER_AUTH_USER_PASSWORD', "")
 
+# Admin Auth Info
+AUTH_USERNAME = os.environ.get("RANCHER_AUTH_USERNAME")
```

Comment:
@rancher-max in `test_auth` the `enable_ad` method makes use of admin_user which comes from the `v3_api/resource/*.json` files loaded in `test_auth`'s fixture. Do we we really need these variables or are these to set the build-in `admin` user password? @sangeethah

Reply:
It seems the requirement in `test_auth` is to have the `AUTH` (AD/OpenLDAP/FreeIPA) admin already setup in the Rancher server, and these two variables are specific for Istio. Please correct me if I'm wrong.

Reply:
We would definitely need at least password. 

Admin user could technically come from the `*.json` files but I wanted to allow it to be managed differently since our internal document for enabling AD auth setup uses a different admin user than the one noted in `v3_api/resource/activedirectory.json`

The other problem was that `test_auth.py` requires admin user and other auth users to have the same password, which I don't think is the case anymore for at least our AD setup so I needed to allow them to be separate.

Reply:
I made this better now actually so it is just the password!

Reply:
@rancher-max 
I saw this line multiple times when enabling the auth
```
AUTH_ADMIN_USER = load_setup_data()["admin_user"]
```

So we are not using this global variable `AUTH_ADMIN_USERNAME`? reserved for future use?

Reply:
We could use that. This way grabs it from the *.json file. It might make more sense to grab it and then store it to use later